app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -131,7 +134,14 @@ class TabTrayDialogFragment : AppCompatDialogFragment() {
             startingInLandscape = requireContext().resources.configuration.orientation ==
                     Configuration.ORIENTATION_LANDSCAPE,
             lifecycleScope = viewLifecycleOwner.lifecycleScope
-        ) { tabsFeature.get()?.filterTabs(it) }
+        ) { private ->
+            val filter: (TabSessionState) -> Boolean =
+                if (private) { state -> state.content.private } else { state -> !state.content.private }
```

Comment:
I think this is easier to read as `private == state.content.private`

Reply:
tru